libs/core/src/components/pds-combobox/stories/pds-combobox.stories.js

```diff
@@ -74,9 +84,69 @@ ButtonTrigger.args = {
   triggerWidth: '250px',
 };
 
+export const ChipTriggerAndLayout = (args) => html`
+<pds-box min-width="100%" direction="row" gap="sm" wrap>
+  <pds-combobox
+    component-id=${args.componentId}
+    label=${args.label}
+    placeholder=${args.placeholder}
+    trigger=${args.trigger}
+    mode=${args.mode}
+    custom-option-layouts=${args.customOptionLayouts}
+    custom-trigger-content=${args.customTriggerContent}
+    trigger-width=${args.triggerWidth}
+  >
+    <pds-chip slot="trigger-content" sentiment="neutral" variant="dropdown" dot>Draft</pds-chip>
```

Comment:
Do you know if the chip always has to match the selected option here? Meaning if the draft chip is displayed by default, then they select a different option, when saving/reloading, they would need to update this chip to match the selected option?

I feel like this could be handled programmatically, though I could be wrong. The chip in the markup feels like we should only need it if there is no option selected; otherwise, the selected option would be displayed. Which may not be a realistic use case, since it feels like, for the chip, an option should always be selected. 

Not a blocker.